layouts/ListLayout.tsx

```diff
@@ -1,85 +1,176 @@
+    initialDisplayPosts.length > 0 && !searchValue
+      ? initialDisplayPosts
+      : filteredBlogPosts
 
   return (
     <>
-      <div className="divide-y">
-        <div className="pt-6 pb-8 space-y-2 md:space-y-5">
-          <h1 className="text-3xl font-extrabold leading-9 tracking-tight text-gray-900 dark:text-gray-100 sm:text-4xl sm:leading-10 md:text-6xl md:leading-14">
-            {title}
+      <div className="divide-y divide-border">
+        <div className="space-y-2 pb-8 pt-6 md:space-y-5">
+          <h1 className="text-3xl font-semibold leading-9 tracking-tight text-foreground sm:text-4xl sm:leading-10 md:text-6xl md:leading-14">
+            {title && title}
           </h1>
-          <div className="relative max-w-lg">
-            <input
-              aria-label="Search articles"
-              type="text"
-              onChange={(e) => setSearchValue(e.target.value)}
-              placeholder="Search articles"
-              className="block w-full px-4 py-2 text-gray-900 bg-white border border-gray-300 rounded-md dark:border-gray-900 focus:ring-primary-500 focus:border-primary-500 dark:bg-gray-800 dark:text-gray-100"
-            />
-            <svg
-              className="absolute w-5 h-5 text-gray-400 right-3 top-3 dark:text-gray-300"
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              stroke="currentColor"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                strokeWidth={2}
-                d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+          <div className="relative w-full">
+            <label>
+              <span className="sr-only">Search articles</span>
+              <input
+                aria-label="Search articles"
+                type="text"
+                onChange={(e) => setSearchValue(e.target.value)}
+                placeholder="Search articles"
+                className="block w-full rounded-md border border-border bg-muted px-4 py-2 text-muted-foreground placeholder-muted-foreground focus:border-primary focus:ring-primary"
               />
-            </svg>
+            </label>
+            <Search className="absolute right-3 top-3 h-5 w-5 text-muted-foreground" />
           </div>
         </div>
-        <ul>
-          {!filteredBlogPosts.length && 'No posts found.'}
-          {displayPosts.map((frontMatter) => {
-            const { slug, date, title, summary, tags } = frontMatter
+        <ul className="pt-4">
+          {!filteredBlogPosts.length && (
+            <div className="mt-4 text-muted-foreground">No posts found.</div>
+          )}
+          {displayPosts.map((post) => {
+            const { path, date, title, authors, summary, tags } = post
+            const authorSlug =
+              authors && authors.length > 0 ? authors[0] : undefined
```

Comment:
Is it intentional to abandon the multi-author data?

Reply:
I forgot to implement multi-author data because we simply haven't had multiple authors before. Good catch, I'll add it to both the homepage's modified layout and `ListLayout`

Reply:
> I forgot to implement multi-author data because we simply haven't had multiple authors before. Good catch, I'll add it to both the homepage's modified layout and `ListLayout`

https://sekai.team/blog/ugra-ctf-quals-2022/snekpeek/ Does this not count as multi-author?

Reply:
> Does this not count as multi-author?

It does, I didn't know about this post at all actually. Thank you

> Also I do see [sekai-team.vercel.app/blog/ugra-ctf-quals-2022/snekpeek](https://sekai-team.vercel.app/blog/ugra-ctf-quals-2022/snekpeek) renders it successfully

Yeah it renders in `PostLayout` but not in `ListLayout`, you can see that I use `authors[0]` (primary author) on `ListLayout` rather than rendering all authors

Reply:
This is resolved in the next commit.